packages/webapp/src/components/Task/AbandonTask/index.jsx

```diff
@@ -83,19 +94,60 @@ const PureAbandonTask = ({
 
       <Main style={{ marginBottom: '24px' }}>{t('TASK.ABANDON.WHEN')}</Main>
 
-      <Input
-        label={t('TASK.ABANDON.DATE')}
-        hookFormRegister={register(ABANDON_DATE, {
-          required: true,
-          validate: isNotInFuture,
-        })}
-        errors={errors[ABANDON_DATE] ? isNotInFuture() : null}
-        style={{ marginBottom: '24px' }}
-        type={'date'}
-        max={getDateInputFormat()}
-        required
+      <RadioGroup
+        hookFormControl={control}
+        onChange={(e) => setSelectedAbandonOption(e.target.value)}
+        name={ABANDON_DATE_SELECTED}
+        disabled={false}
+        style={{ paddingBottom: '16px' }}
+        radios={[
```

Comment:
@gursimran-singh 

If you could put these into a const above that makes the render more readable. See options in the CreateUserAccount component for examples.